test/e2e/framework.go

```diff
@@ -69,6 +69,8 @@ var AntreaConfigMap *corev1.ConfigMap
 
 var (
 	connectionLostError = fmt.Errorf("http2: client connection lost")
+	// Initialize random number generator with a seed
+	ran = rand.New(rand.NewPCG(uint64(time.Now().UnixNano()), 0))
```

Comment:
why is this needed? why can't we use the main random number generator which is randomly seeded and is also thread-safe?